### PR TITLE
feat: multi public-key for account delete-key command

### PR DESCRIPTION
**Pull request by Mehrbod2002**
Fixes : #205 
multi public-key for account delete-key command separated  by `,` .

### CODE REVIEW

**Comment by frol** [comment]
@Mehrbod2002 Thanks for the contribution!

@FroVolod Please, review this PR. It seems that it does not have checkbox-list in the interactive mode and instead asks a user to specify public keys as a comma-separated list, but it might be already good enough for the time being while checkbox-list can be implemented later.

File: src/commands/account/delete_key/use_publickeylist_type/mod.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+use interactive_clap::ToCli;
+
+#[derive(Debug, Clone)]
+pub struct PublicKeyList(pub Vec<crate::types::public_key::PublicKey>);
```
Comment (frol) [inline src/commands/account/delete_key/use_publickeylist_type/mod.rs:4]: I think you can use `near_crypto::PublicKey` here directly. There is no point in using the wrapper type. And then you don't need to repackage the items at all: https://github.com/near/near-cli-rs/pull/207/files#diff-a637fdd677d48fb8d9a4460b98e3ed6a82e647bebbf16435e42e1e667f829ef0R35-R37

File: src/commands/account/delete_key/mod.rs [hunk 2]
```diff
@@ -26,23 +28,36 @@ impl DeleteKeyCommandContext {
...
             config: previous_context.0,
             owner_account_id: scope.owner_account_id.clone().into(),
-            public_key: scope.public_key.clone().into(),
+            public_keys: scope
+                .public_keys
+                .clone()
+                .0
+                .into_iter()
+                .map(Into::into)
+                .collect(),
```
Comment (frol) [inline src/commands/account/delete_key/mod.rs:37]: See another comment.

```suggestion
```

File: src/commands/account/delete_key/mod.rs [hunk 3]
```diff
@@ -26,23 +28,36 @@ impl DeleteKeyCommandContext {
...
         })
     }
 }
 
 impl From<DeleteKeyCommandContext> for crate::commands::ActionContext {
     fn from(item: DeleteKeyCommandContext) -> Self {
+        let public_keys_clone = std::sync::Arc::new(item.public_keys.clone());
```
Comment (frol) [inline src/commands/account/delete_key/mod.rs:44]: Drop the Arc, cloning is enough.

```suggestion
        let public_keys = item.public_keys.clone();
```

**Review by frol — CHANGES_REQUESTED** [review]

File: src/commands/account/delete_key/use_publickeylist_type/mod.rs [hunk 4]
```diff
@@ -0,0 +1,32 @@
+use interactive_clap::ToCli;
```
Comment (frol) [inline src/commands/account/delete_key/use_publickeylist_type/mod.rs:1]: Please, move this file to `src/types/public_key_list.rs`

File: src/commands/account/delete_key/mod.rs [hunk 5]
```diff
@@ -26,23 +28,36 @@ impl DeleteKeyCommandContext {
...
-                            public_key: item.public_key.clone(),
-                        },
-                    )],
+                    actions: public_keys_clone
+                        .clone()
+                        .iter()
+                        .map(|public_key| {
+                            near_primitives::transaction::Action::DeleteKey(
+                                near_primitives::transaction::DeleteKeyAction {
+                                    public_key: public_key.clone(),
```
Comment (frol) [inline src/commands/account/delete_key/mod.rs:56]: I believe this should work and void unnecessary cloning:

```suggestion
                        .into_iter()
                        .map(|public_key| {
                            near_primitives::transaction::Action::DeleteKey(
                                near_primitives::transaction::DeleteKeyAction {
                                    public_key,
```

**Comment by Mehrbod2002** [comment]
@frol Now you can review it .

**Review by frol — APPROVED** [review]
Looks good to me now. Waiting for CI and ready to merge!

**Pull request merged**